services/openuserjs/openuserjs-license.service.js

```diff
@@ -0,0 +1,39 @@
+import { InvalidResponse } from '../index.js'
+import { renderLicenseBadge } from '../licenses.js'
+import BaseOpenUserJSService from './openuserjs-base.js'
+
+export default class OpenUserJSLicense extends BaseOpenUserJSService {
+  static category = 'license'
+  static route = { base: 'openuserjs', pattern: 'l/:username/:scriptname' }
+
+  static examples = [
+    {
+      title: 'OpenUserJS',
+      namedParams: {
+        username: 'NatoBoram',
+        scriptname: 'YouTube_Comment_Blacklist',
+      },
+      staticPreview: renderLicenseBadge({ licenses: ['GPL-3.0-or-later'] }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'license' }
+
+  transform(data) {
+    if (!('license' in data.UserScript)) {
+      throw new InvalidResponse({
+        prettyMessage: 'license not found',
+      })
```

Comment:
@DenverCoder1 

One more quickie... sorry... if it is absent, say at https://openuserjs.org/scripts/JamesSwift/Newegg_ComboEgger , we apply `MIT` as the TOS default.... so not sure if you want to throw an actual error or put in a static `MIT`. Absentees no longer occur but older scripts weren't enforced.

Reply:
Done :+1: